source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,266 @@
+.. _aws-web:
+
+Create and launch Clear Linux from Amazon Web Services
+######################################################
+
+This tutorial walks you through creating and launching a |CL|
+:abbr:`AMI (Amazon Machine Image)` instance from the
+:abbr:`AWS (Amazon Web Services)` console and complete the following tasks:
+
+#. Locate and select the |CL| OS Basic AMI in the AWS Marketplace.
+#. Create a new public and private key pair to allow you to connect to your
+   |CL| instance securely.
+#. Launch the new |CL| instance and connect to it.
+#. Update your instance of |CL| using the :command:`swupd` command.
+#. Stop the |CL| instance.
+
+Prerequisites
+*************
+
+This tutorial assumes the following:
+* You are using a linux-based system to access AWS and can run :command:`SSH`
+  to access the remote |CL| AWS image.
+* Your browser puts downloaded files in the :file:`$HOME/Downloads` directory.
+* You have already set up an AWS user account and logged in to the AWS
+  console.
+  
+.. note::
+   This tutorial uses a |CL| AMI t2.micro instance that is eligible for the
+   AWS free tier. To learn more about AWS and setting up an account, visit the
+   AWS website at http://aws.amazon.com.
+
+Locate, select, and launch the Clear Linux OS Basic AMI
+*******************************************************
+
+#. Start from your main AWS console, select :guilabel:`EC2` as shown in
```

Comment:
Missing noun and article. What should I select? A button, a meny, a tab, etc... This issue is found multiple times in the document but I will only comment this once.

Reply:
Changed to:  
#. Start from your main AWS services console menu in your browser and select the
   :guilabel:`EC2` text as shown in figure 1: